apps/judicial-system/backend/src/app/modules/case-table/caseTable.cellGenerators.ts

```diff
@@ -402,6 +407,156 @@ const prisonAdminState: CaseTableCellGenerator = {
   },
 }
 
+const indictmentAppealDeadline: CaseTableCellGenerator = {
+  attributes: ['rulingDate', 'indictmentRulingDecision'],
+  generate: (c: Case): StringGroupValue | undefined => {
+    if (!c.rulingDate) {
+      return undefined
+    }
+
+    const indictmentAppealDeadline = formatDate(
+      getIndictmentAppealDeadlineDate(
+        c.rulingDate,
+        c.indictmentRulingDecision === CaseIndictmentRulingDecision.FINE,
+      ),
+    )
+
+    if (!indictmentAppealDeadline) {
+      return undefined
+    }
+
+    return { s: [indictmentAppealDeadline] }
+  },
+}
+
+const subpoenaServiceState: CaseTableCellGenerator = {
+  attributes: ['rulingDate', 'indictmentRulingDecision'],
+  includes: {
+    defendants: {
+      model: Defendant,
+      attributes: ['serviceRequirement', 'verdictViewDate'],
+      order: [['created', 'ASC']],
+      separate: true,
+    },
+  },
+  generate: (c: Case): TagValue | undefined => {
+    const isFine =
+      c.indictmentRulingDecision === CaseIndictmentRulingDecision.FINE
+
+    if (isFine) {
+      return undefined
+    }
+
+    const isRuling =
+      c.indictmentRulingDecision === CaseIndictmentRulingDecision.RULING
+    const verdictInfo = c.defendants?.map<[boolean, Date | undefined]>((d) => [
+      isRuling || isFine,
```

Comment:
Nit: `isFine` is unrelevant here right? It will always be `false`. Same below

Reply:
Yes, I know. This code is copied directly from a transformer in the api layer and I intended to rewrite this at some point. I apparently forgot and will take a look.